internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -612,7 +609,7 @@ func resourceCdnFrontDoorFirewallPolicyRead(d *pluginsdk.ResourceData, meta inte
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorFirewallPolicyID(d.Id())
+	id, err := parse.FrontDoorFirewallPolicyIDInsensitively(d.Id())
```

Comment:
this is coming from the config and so should be case-sensitive

```suggestion
	id, err := parse.FrontDoorFirewallPolicyID(d.Id())
```